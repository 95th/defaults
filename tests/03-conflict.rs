use defaults::Defaults;

#[derive(Defaults, Default)]
struct Foo {
    x: usize,
    y: usize,
}

fn main() {}
