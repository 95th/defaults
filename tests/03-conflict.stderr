error[E0119]: conflicting implementations of trait `std::default::Default` for type `Foo`:
 --> $DIR/03-conflict.rs:3:10
  |
3 | #[derive(Defaults, Default)]
  |          ^^^^^^^^  ------- first implementation here
  |          |
  |          conflicting implementation for `Foo`
